### DISCUSSION TITLE
Request: versioned libraries for Fedora/EPEL packages

### DISCUSSION THREAD

**Discussion by neelchauhan** [accel-ppp/accel-ppp#212] [Development discussions]
I'm the package maintainer for accel-ppp on Fedora, well, just created it, and as a norm, Fedora asks for all packages to be versioned.

https://docs.fedoraproject.org/en-US/packaging-guidelines/#_downstream_so_name_versioning

> In cases where upstream ships unversioned .so library (so this is not needed for plugins, drivers, etc.), the packager MUST try to convince upstream to start versioning it.

Could you please version the libraries in accel-ppp 1.14.0 or 1.13.1 or whatever the next version is?

**Comment by svlobanov** [comment]
Hi @neelchauhan if you know how to do it, could you please send a PR? 

BTW, which cmake options will you use? Are you going to provide ipoe and vlan-mon kernel modules?

**Comment by marekm72** [comment]
Just a thought - would it be possible to simplify packaging (for any Linux distro, not just Fedora) by linking all these *.so modules statically in the single accel-ppp binary?  The ones not configured to be used would still be there, but their init functions not called.  These shared libraries are internal to accel-ppp (not shared with anything else) anyway, and not large enough for significant memory savings by removing the unused ones.
